service/controller/kvm/v8/version_bundle.go

```diff
@@ -7,13 +7,22 @@ import (
 func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
+			{
+				Component:   "coredns",
+				Description: "Add coredns chartconfig.",
```

Comment:
This message is not very explicit on what adding coredns means, how about:
Add management of CoreDNS component for tenant clusters. Also enable CoreDNS custom configuration.

Reply:
Or align with aws and set same message: `Allow advanced CoreDNS configuration. See https://docs.giantswarm.io/guides/advanced-coredns-configuration`

Reply:
Yes please align with aws.

Reply:
done